lua/calltree/ui.lua

```diff
@@ -345,11 +345,15 @@ end
 --
 -- keep_open : bool - if true, and the panel is open,
 -- the panel will be left open when this function terminates.
+--
+-- returns bool | err: true when panel was opened by this call,
```

Comment:
just for consistency can you make it look like the comment above? 

I actually do not like this pattern and plan on moving all the comments to "@param" documentations like seen in other lua code bases. But for now, I like consistency over anything

Reply:
https://github.com/ldelossa/calltree.nvim/blob/73f336dbd173c3e597c5a8992a8a1913e510472d/lua/calltree/tree/tree.lua#L19

Heres an example of what returns look like right now.

Reply:
force-pushed modified doc strings in `ui.lua` and added doc strings in `window.lua`.